livekit-agents/livekit/agents/voice/agent_session.py

```diff
@@ -116,6 +126,7 @@ def __init__(
         max_endpointing_delay: float = 6.0,
         max_tool_steps: int = 3,
         video_sampler: NotGivenOr[_VideoSampler | None] = NOT_GIVEN,
+        user_away_timeout: NotGivenOr[float] = NOT_GIVEN,
```

Comment:
Just thinking out loud:
```suggestion
        user_inactivity_timeout: NotGivenOr[float] = NOT_GIVEN,
```